app/views/research_sessions/preview.html.erb

```diff
@@ -46,105 +46,84 @@
     </p>
   </section>
 
+  <% methodologies = capture do %>
+    <ul class="bullet-point-list">
+      <% @research_session.methodologies.each do |methodology| %>
+        <li>
+          <%= edit_link_for(:methodologies) do %>
+            <%= methodology_lookup(methodology) %>
+          <% end %>
+        </li>
+      <% end %>
+    </ul>
+  <% end %>
+
+  <% recording = capture do %>
+    <p>
+      Which we will record using:
+      <ul class="bullet-point-list">
+        <% @research_session.recording_methods.each do |recording_method| %>
+          <li>
+            <%= edit_link_for(:recording_methods) do %>
+              <%= recording_method_lookup(recording_method) %>
+            <% end %>
+          </li>
+        <% end %>
+      </ul>
+    </p>
+  <% end %>
+
   <%=
     react_component(
-      'ResearcherPreviews/WhoIsDoingTheResearch',
-      component_params(
-        :researcher_job_title, :researcher_name, :researcher_other_name,
-        final_preview: true
-      ),
+      'Shared/WhatHappensInThisResearchSession',
+      component_params(:researcher_job_title, :researcher_name, final_preview: true).merge({
+        methodologies_markup: methodologies,
+        recording_markup: recording,
+        you_or_your_child: you_or_your_child
+      }),
       {
         prerender: true
       }
     )
   %>
 
   <section>
-    <h3 class="subtitle-small" id="what">What will happen next?</h3>
-
-    <% @research_session.methodologies.each do |methodology| %>
+    <h3 class="subtitle-small" id="session-details">Session details</h3>
+    <% if @research_session.where_when_enabled %>
+      <p>The session is</p>
+      <dl class="definition-list">
+        <%
+          {
+            when_text: 'On',
+            location:  'At',
+            duration:  'Duration'
+          }.each_pair do |attr, term|
+        %>
+          <% if @research_session.send(attr).present? %>
+            <div>
+              <dt><%= term %>:</dt>
+              <dd><%= edit_link_for(attr) %></dd>
+            </div>
+          <% end  %>
+        <% end %>
+      </dl>
       <p>
-        <%= edit_link_for(:methodologies) do %>
-          <%= methodology_lookup(methodology) %>
+        <%= your_or_your_childs.capitalize %> time and what <%= you_or_they %>
```

Comment:
This sentence should only be visible when incentives enabled. If no incentive is provided, there's no point in saying that what you say is important..